FND/Accounting/AccountingEquity.rdf

```diff
@@ -68,30 +68,31 @@
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20160201/Accounting/AccountingEquity.rdf version of this ontology was modified per the FIBO 2.0 RFC to rework definitions in support of revised Business Entities, Equities and related ontologies.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20180801/Accounting/AccountingEquity.rdf version of this ontology was modified to eliminate deprecated elements.</skos:changeNote>
 		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20190901/Accounting/AccountingEquity.rdf version of this ontology was modified to augment the definition of asset, and add income.</skos:changeNote>
+		<skos:changeNote>The https://spec.edmcouncil.org/fibo/ontology/FND/20200101/Accounting/AccountingEquity.rdf version of this ontology was modified to add physical asset and eliminate ambiguity in some definitions.</skos:changeNote>
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
 	<owl:Class rdf:about="&fibo-fnd-acc-aeq;Capital">
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-acc-aeq;takesForm"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
 				<owl:allValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>capital</rdfs:label>
-		<skos:definition>money and other property of a corporation or other enterprise used in transacting its business</skos:definition>
+		<skos:definition>money and other property of an organization used in transacting its business</skos:definition>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-fnd-acc-aeq;CapitalSurplus">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-acc-aeq;PaidInCapital"/>
 		<rdfs:label>capital surplus</rdfs:label>
-		<skos:definition>capital contributed in excess of of the par value (stated value) of the ownership interest issued</skos:definition>
+		<skos:definition>capital contributed in excess of the par value (stated value) of the ownership interest issued</skos:definition>
```

Comment:
That's pretty opaque. "contributed" by whom?
It's more complex anyway https://www.investopedia.com/terms/c/capitalsurplus.asp#:~:text=Capital%20surplus%20includes%20equity%20or,referred%20to%20as%20capital%20surplus. 
Note that this article says "Most balance sheets today call capital surplus paid-in surplus or paid-in capital [in excess of par]." so it should be a synonym not a subclass.

Reply:
Actually not according to my SME, who has been a bank auditor and works in this area daily.  The definition is correct.  Investopedia is not definitive on this.  And, we are not necessarily talking exclusively about balance sheets.

Reply:
... which is why a use case would help

Reply:
Regardless of the synonym, the definition could use some clarifying.

Reply:
Barron's has a fairly opaque definition - I'll see what I can find elsewhere.